lib/kitto/router.ex

```diff
@@ -101,4 +105,12 @@ defmodule Kitto.Router do
   end
 
   defp default_dashboard, do: Application.get_env(:kitto, :default_dashboard, "sample")
+
+  defp authenticate(conn) do
```

Comment:
This seems like a good candidate for a predicate function, since it'll always return true || false like:

```elixir
defp authenticated?(conn)
```